assets/sass/utilities/_utilities-accents.scss

```diff
@@ -1,14 +1,9 @@
 /* =========================================================================
    Utilities: Tones & Accents
    ========================================================================= */
-// Utility classes to help with colour accent theming
-
-/* =========================================================================
-   Tones
-   ========================================================================= */
 
 /* Foreground tones */
-.u-tone-background-brand-primary {
```

Comment:
The class name included **background** but it changed the **foreground** colour. 🤦‍♂️